src/main/kotlin/no/nav/syfo/sykmelding/integration/aktor/client/AktoerIdClient.kt

```diff
@@ -46,6 +47,7 @@ class AktoerIdClient(
         val patientIdents = aktoerIds[fnr]
 
         if (patientIdents == null || patientIdents.feilmelding != null) {
+            EGENMELDT_SYKMELDING_FAILED_COUNTER.inc()
```

Comment:
hmm, her kommer det kanskje litt an på om vi ønsker å måle funksjonelle feil eller tekniske feil. Jeg tenker at denne counteren her måler tekniske feil, og i så fall bør den kanskje telle når vi treffer inni her? https://github.com/navikt/egenmeldt-sykmelding-backend/blob/master/src/main/kotlin/no/nav/syfo/application/ApplicationEngine.kt#L70

Vi kan gjerne lage en egen counter for funksjonelle feil der vi har en egen type for hver funksjonell feil som kan oppstå. Den bør i så fall trigges i den errorhandleren som settes opp på linjen over den jeg linket til.

Reply:
Her er et eksempel som bruker type: https://github.com/navikt/syfovarsel/blob/master/src/main/java/no/nav/syfo/util/Metrikk.java#L24

Hvis det er vanskelig å få til type kan vi også bare lage en metrikk pr funksjonelle feiltype :)

Reply:
Ah, jeg så ikke at det var en global catch for feil. Takk!

Jeg tenker kanskje at for en slik tjeneste som dette, hvor levetid trolig er ganske kort (uker/måneder), så holder det kanskje å telle antall forespørsler som feiler totalt sett?

Reply:
Nja, vi vet jo ikke om den varer kort da? Jeg kunne godt tenke meg å se om det er mange som blir avvist pga at de har sykmelding fra før f.eks. (når vi implementerer regel for det), det er jo den eneste måten vi kan lære noe om brukerne her :)

Reply:
Jeg har ikke fått inntrykk av at muligheten for å sykmelde seg selv er ment å være permanent, så jeg regner det som kortvarig. Men hvis det er interessant å logge mer detaljert, uavhengig av varighet, så legger jeg det til :)

Reply:
Jeg tenker det kan være interessant ja, for å se hvor godt det treffer :)